six/assetlist.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "six",
+    "assets": [
+      {
+        "description": "The native staking token of the SIX Protocol.",
+        "denom_units": [
+          {
+            "denom": "usix",
+            "exponent": 0
+          },
+          {
+            "denom": "six",
+            "exponent": 6
+          }
+        ],
+        "base": "usix",
+        "name": "Six token",
+        "display": "six",
+        "symbol": "six",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/sixtestnet/images/six.png"
```

Comment:
This image provided might make an appropriate chain logo (which, currently there is no chain logo defined), but isn't as appropriate for asset logo due to not having square aspect ratio and the text ('SIX Protocol') at the bottom. Perhaps we mean to add another copy of the image but without the text?